catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImplTest.java

```diff
@@ -75,8 +88,14 @@
 
   @Mock private WorkspaceMetacardFilter workspaceMetacardFilter;
 
+  @Mock private FilterBuilder filterBuilder;
+
+  @Mock private QueryResponse queryResponse;
+
   @Before
   public void setup() {
+    filterBuilder = mock(FilterBuilder.class, RETURNS_DEEP_STUBS);
```

Comment:
You can configure this using the `@Mock` annotation: `@Mock(answer = RETURNS_DEEP_STUBS)`